Anu2g/redux-persist
# Dynamic Whitelist

## General Info

1. Purpose: Pass dynamic entity key set to redux persist so we can reduce the number of edits we need to make to the fork. Allow a list of entities to be passed to redux persist and then immediately flushed to the disk.
2. Effects on User: Better entity storage management
3. User segments affected: All

### Related PRs:
- https://github.com/Hurdlr/TraxData/pull/514
- https://github.com/Hurdlr/TraxReactNative/pull/3402

## Test Info
### Test Scenarios

1. Login to an account
1. Make several edits
1. Background app
1. Foreground app
1. Verify edits persisted
1. Make more edits
1. Kill app
1. Foreground app
1. Verify edits persisted
